src/Aspire.Hosting/Dcp/ApplicationExecutor.cs

```diff
@@ -780,10 +780,19 @@ private async Task StartDashboardAsDcpExecutableAsync(CancellationToken cancella
             KeyValuePair.Create(DashboardConfigNames.ResourceServiceUrlName.EnvVarName, resourceServiceUrl),
             KeyValuePair.Create(DashboardConfigNames.DashboardOtlpUrlName.EnvVarName, otlpEndpointUrl),
             KeyValuePair.Create(DashboardConfigNames.ResourceServiceAuthModeName.EnvVarName, "Unsecured"),
-            KeyValuePair.Create(DashboardConfigNames.DashboardFrontendAuthModeName.EnvVarName, "Unsecured"),
         };
 
-        if (configuration["AppHost:OtlpApiKey"] is { } otlpApiKey)
+        if (configuration["AppHost:BrowserToken"] is { Length: > 0 } browserToken)
```

Comment:
Should we use IOptions<T> for this.

Reply:
I agree. I didn't want to rewrite the host in this PR.